app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java

```diff
@@ -2,14 +2,21 @@
 
 import android.support.annotation.NonNull;
 import android.support.v7.widget.RecyclerView;
+import android.util.Patterns;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+
 import org.fossasia.susi.ai.R;
 import org.fossasia.susi.ai.chat.adapters.viewholders.VerticalCellViewHolder;
+
```

Comment:
Is this extra line needed?

Reply:
it separates the type of dependencies that they are viewed better